README.md

```diff
@@ -10,7 +10,7 @@
 
 Pywbem is a WBEM client and WBEM indication listener and provides
 related WBEM client-side functionality. It is written in pure Python and
-runs on Python 2 and Python 3.
+runs on Python 3.
```

Comment:
Should we be defining the minimum python version here?

Reply:
No. It just creates maintenance effort.